addons/l10n_se_K2_mini/data/account_fiscal_position_tax_template.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- ************************************************************* -->
+<!-- Fiscal Position Tax Templates -->
+<!-- ************************************************************* -->
+<odoo>
+	<data noupdate="1">
+		<!-- Fiscal Position Sweden -->
+		<record id="fpp_sweden_25_services" model="account.fiscal.position.tax.template">
+			<field name="position_id" ref="fp_sweden" />
+			<field name="tax_src_id" ref="purchase_tax_25_services" />
```

Comment:
I'm suprised you map a service account onto a goods account. Are you sure about this ? How come ?

Reply:
I will remove it since it doesn't do anything, both service and goods in Sweden uses the same account. However it uses different accounts for EU or outside EU.